crates/dt-workspace/src/config/toml_config.rs

```diff
@@ -0,0 +1,48 @@
+use serde::{Deserialize, Serialize};
+use std::path::{Path, PathBuf};
+use thiserror::Error;
+
+pub const CONFIG_FILENAME: &str = ".dt-tools.toml";
+
+#[derive(Debug, PartialEq, Default, Serialize, Deserialize)]
+#[serde(rename_all = "kebab-case")]
+/// Workspace configuration in TOML format
+///
+/// **Note**: It's up to the caller to perform validation after deserialization.
```

Comment:
Forgot this in the earlier PR, but doc-comments always go before macros.
```suggestion
/// Workspace configuration in TOML format
///
/// **Note**: It's up to the caller to perform validation after deserialization.
#[derive(Debug, PartialEq, Default, Serialize, Deserialize)]
#[serde(rename_all = "kebab-case")]
```